BookmarksTodayExtension/Info.plist

```diff
@@ -5,7 +5,7 @@
 	<key>CFBundleDevelopmentRegion</key>
 	<string>en</string>
 	<key>CFBundleDisplayName</key>
-	<string>DuckDuckGo</string>
+	<string>DuckDuckGo - Bookmarks</string>
```

Comment:
Maybe we could have a shorter name? With show more/show less button visible it doesn't look that good. :)

Reply:
This came from the designs. I'll propose it to design and add it as a potential future tweak.